start/data/osgconnect-storage.md

```diff
@@ -12,21 +12,65 @@
+| **StashCache**        |  < 1 GB   | stashcp | Transfer data to stash|  [StashCache](https://support.opensciencegrid.org/support/solutions/articles/12000002775) |
+| **GridFTP**  |  > 1 GB, < 50 GB    | gfal-copy | Output data to ~/stash| Experts with large work flows. Please contact us if you want to use this.|
+
+
+### External data transfer
+
+|  | **Data Size**| **Tools** |**Details**|
+|:------------|:-------:|:------|:------| 
+|**UNIX tools** | < 1 GB | rsync, scp, putty, WinSCP, gFTP, etc.  |[SCP](https://support.opensciencegrid.org/support/solutions/articles/5000634376) |
+|**Globus** |  > 1 GB  | globus webservice or globus CLI | [Globus](https://support.opensciencegrid.org/support/solutions/articles/5000632397) |
+
+
+## Storage Options
+### home
+Home is meant for storing files long-term. Usually, files such as source code, parameter files, scripts, etc. are kept in your `/home` directory. The disk quota on home is 20 GBs. When a user exceeds his quota, the system will send email notifications. If the notice is disregarded, eventually the user will lose the privilege to write on his home.
+
+** Home filesystem is not suitable to run your HTCondor jobs. It is a good practice to run all your jobs under the `local-scratch` or `stash` directories. **
+
+### local-scratch
 `local-scratch` is meant for temporary storage, and is implemented as fast local disk for each submit node. It is a good practice to run your jobs on `local-scratch` and move the output data to a secondary local disk as soon as possible. NOTE: data on `local-scratch` is automatically removed after 30 days and is not backed up.
 
-## stash
-Stash provides medium term storage (less 6 months) for users. Stash is a distributed shared filesystem mounted on all our submit nodes, and is therefore slower than local-scratch. There is no disk quota imposed, nor a automatic purge, on stash. However, the data on stash is not backed up, so you should regularly transfer your data on stash to a long term archival system.  If you are transfering a large amount of data, please use the Globus transfer service.  For more details check the articles on [data transfer techniques](https://support.opensciencegrid.org/solution/folders/5000260918).  
+### stash
+Stash provides medium term storage (less 6 months) for users. Stash is a distributed shared filesystem mounted on all our submit nodes, and is therefore slower than local-scratch. There is no disk quota imposed, nor a automatic purge, on stash. However, the data on stash is not backed up, so you should regularly transfer your data on stash to a long term archival system.  If you are transfering a large amount of data, please use the Globus transfer service.  For more details check the articles on [data transfer techniques](https://opensciencegrid.freshdesk.com/a/solutions/folders/12000013267).  
```

Comment:
(less 6 months) --> (less than 6 months)

This isn't actually true, so I'm not sure it should go in the guide until we're going to enforce it.